app/src/main/index.js

```diff
@@ -130,31 +130,7 @@ function createWindow () {
   webContents.on('will-navigate', handleRedirect)
   webContents.on('new-window', handleRedirect)
 
-  // setup menu to handle copy/paste, etc
-  var template = [
```

Comment:
why are you removing this?

Reply:
This removes the menu: https://github.com/cosmos/cosmos-ui/issues/236